R/achillesCodeUse.R

```diff
@@ -12,6 +12,14 @@
 #' @export
 #'
 #' @examples
+#' \dontrun{
+#' cdm <- mockVocabRef("database")
+#' oa <- getCandidateCodes(cdm = cdm, keywords = "osteoarthritis")
+#' result_achilles <- achillesCodeUse(list(oa = oa$concept_id), cdm = cdm)
+#' result_achilles
+#' DBI::dbDisconnect(attr(cdm, "dbcon"), shutdown = TRUE)
```

Comment:
@xihang-chen can you use cdm_disconnect here (https://odyosg.github.io/CDMConnector/reference/cdmDisconnect.html)?